virttest/qemu_vm.py

```diff
@@ -463,12 +463,13 @@ def add_human_monitor(devices, monitor_name, filename):
                 return " -monitor unix:'%s',server,nowait" % filename
 
             monitor_id = "hmp_id_%s" % monitor_name
-            cmd = " -chardev socket"
-            cmd += _add_option("id", monitor_id)
-            cmd += _add_option("path", filename)
-            cmd += _add_option("server", "NO_EQUAL_STRING")
-            cmd += _add_option("nowait", "NO_EQUAL_STRING")
-            cmd += " -mon chardev=%s" % monitor_id
+            chardev_params = params.object_params(monitor_name)
+            # only support default chardev_backend 'unix_socket' for hmp monitor
+            params["monitor_filename_%s" % monitor_name] = filename
+            char_device = devices.chardev_define_by_params(
```

Comment:
do we need to  force update chardev_backend_type = unix_socket in here?

Reply:
It is, but i did not set it here, since: https://github.com/avocado-framework/avocado-vt/blob/814de27e80573c5117d8068b98e1f97d95568bc6/virttest/qemu_devices/qcontainer.py#L2296
I am ok to force set it here again. What's your opinion ?

Reply:
better to check backend type here,  I suggest to raise exception if backend  is not unix_socket.

Reply:
updated, thanks!